backend/pipeline/views.py

```diff
@@ -110,3 +118,26 @@ def partial_update(self, request: Request, pk: Any = None) -> Response:
             )
         else:
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    @action(detail=True, methods=["get"])
```

Comment:
@muhammad-ali-e is this required? we are giving this in urls rit?

Reply:
It's there in the APIDeployment also. I think It's for API docs to show the details.